source/agora/common/Config.d

```diff
@@ -102,6 +102,10 @@ public struct NodeConfig
     static assert(!hasUnsharedAliasing!(typeof(this)),
         "Type must be shareable accross threads");
 
+    /// If set, a commons budget address to use
+    /// in place of the built-in commons budget address as defined by CoinNet
+    public PublicKey commons_budget_address;
```

Comment:
Does it actually  work with string, or does it expect an array of ubyte ?

Reply:
At first, it was declared a string.
But I judged that it would be safer to do verification by converting from this into a structure than to deliver it internally without its verification. And it has data in the ubyte array inside.

Reply:
Nevermind, I misread the diff L337. This works fine 👌